frontend/src/ProfilePageComp/SavedRecordings.jsx

```diff
@@ -27,24 +29,64 @@ function SavedRecordings() {
           const audioBlob = convertToBlob(item.audios.data, "audio/mp4"); //converts each value in data.audios.data into a blob
           return URL.createObjectURL(audioBlob); //creates URL from blob
         });
-        setRecordings(urls);//Updates state variable with recent list of recordings
+        const ids = data.map((item) => {
+          const numID = item.id;
+          return numID;
+        });
+        setRecordings(urls);
+        setRecordingIDs(ids);
       }
     } catch (error) {
       alert("Failed to fetch audios. Please try again.");
     }
   };
 
+  const handleDelete = async (index) => {
+    try {
+      const audioID = recordingIDs[index]
+      const audioURL = recordings[index]
+      await fetch(
```

Comment:
Still want to have error handling here in case the request fails. This applies to all CRUD operations